perma_web/conftest.py

```diff
@@ -270,11 +270,7 @@ class PayingRegistrarFactory(RegistrarFactory):
     cached_paid_through = GENESIS
     base_rate = Decimal(100.00)
     in_trial = False
-
-
-@register_factory
-class PayingLimitedRegistrarFactory(PayingRegistrarFactory):
-    unlimited = False
+    unlimited=True
```

Comment:
Totally minor linting point but maybe 
```suggestion
    unlimited = True
```
for consistency?